routers/repo/issue_dependency_remove.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
Why are these two functions in separate files?

Reply:
Good question. I think simply because I've started with one and made the otherone afterwards...
Should I change it?

Reply:
Yes please 🙂

Reply:
Done.